Source/Engine/Platform/Windows/WindowsWindow.cpp

```diff
@@ -1178,7 +1178,7 @@ LRESULT WindowsWindow::WndProc(UINT msg, WPARAM wParam, LPARAM lParam)
         }
         OnLostFocus();
         break;
-    case WM_ACTIVATEAPP:
+    case WM_ACTIVATE:
```

Comment:
Forgive my absolute noob question, but what's the difference between the two?
WM_ACTIVATE seems to have a 3rd possible value `WA_CLICKACTIVE 2 Activated by a mouse click.`, which should probably also be handled?